src/tiledimage.js

```diff
@@ -2218,11 +2197,18 @@ $.extend($.TiledImage.prototype, $.EventSource.prototype, /** @lends OpenSeadrag
                     // add reference also to the main cache, no matter what the other tile state has
                     // completion of the invaldate event should take care of all such tiles
                     const targetMainCache = t.getCache();
-                    tile.addCache(t.cacheKey, () => {
-                        $.console.error("Attempt to share main cache with existing tile should not trigger data getter!");
-                        return targetMainCache.data;
-                    }, targetMainCache.type, true, false);
+                    tile.setCache(t.cacheKey, targetMainCache, true, false);
                     break;
+                } else if (t.processing) {
+                    console.log("ENCOUNTERED LOADING TILE!!!");
```

Comment:
This should probably be `$.console.log`, right? Also, probably should be reworded?